### PR TITLE
Docs: fix some typos in config.rst

### PR DESCRIPTION
**Pull request by garenchan**
Fix some typos in config.rst

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -53,7 +53,7 @@ In order to set the environment and debug mode reliably, Flask uses
 environment variables.
 
 The environment is used to indicate to Flask, extensions, and other
-programs, like Sentry, what context Flask is running in. It is
+programs, like Sentry, what context they are running in. It is
```
Comment (davidism) [inline docs/config.rst]: This is correct as-is. The paragraph is talking specifically about Flask's environment (`app.env`, `FLASK_ENV`).

File: docs/config.rst [hunk 2]
```diff
@@ -222,7 +222,7 @@ The following configuration values are used internally by Flask:
 
     If ``session.permanent`` is true, the cookie's expiration will be set this
     number of seconds in the future. Can either be a
-    :class:`datetime.timedelta` or an ``int``.
+    :class:`datetime.timedelta` or a ``float``.
```
Comment (davidism) [inline docs/config.rst]: I'd rather these stay as `int`, accepting a `float` is only a side effect of how `timedelta` parses the value, and it's not really useful here.

**Comment by davidism** [comment]
Please make docs changes against 1.0-maintenance (unless they have to do with new features). Otherwise the changes will only appear in the dev docs.

**Comment by garenchan** [comment]
@davidism, do you mean that document related PR should be based on the ``1.0-maintenance`` branch?

**Comment by davidism** [comment]
Yes

**Comment by garenchan** [comment]
There are some docs changes in the dev version that are also useful for ``1.0-maintenance``.  If these commits aren't merged into ``1.0-maintenance``, the next change will be cumbersome.

**Comment by davidism** [comment]
I know, I'm working on cherry picking them.

**Comment by garenchan** [comment]
Thank you for your hard work!

**Pull request merged**